src/libraries/Directory.Build.props

```diff
@@ -314,9 +239,7 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 
     <RuntimePath Condition="'$(RuntimePath)' == ''">$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'runtime', '$(BuildSettings)'))</RuntimePath>
-    <RefRootPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'ref'))</RefRootPath>
```

Comment:
This should remain. It’s used below

Reply:
This needs to be moved bcoz <RefPath> depends on this and we need to import the targetFramework.props before this block of path properties.
If we want to keep <RefRootPath> in this file we need to move this property to just before the import.

Reply:
Don't we need to redefine `BuildTargetFrameworkRefPath` and other properties which rely on `RefRootPath` or transitives?

Reply:
RefRootath is same for all tfms and will be defined before we evaluate BuildTargetFrameworkRefPath.